vendor/github.com/livepeer/go-livepeer-basicnet/basic_out_stream.go

```diff
@@ -85,6 +85,7 @@ func (bs *BasicOutStream) SendMessage(opCode Opcode, data interface{}) error {
 func (bs *BasicOutStream) encodeAndFlush(n interface{}) error {
 	if bs == nil {
 		fmt.Println("stream is nil")
+		return ErrOutStream
```

Comment:
This one is actually already part of a pending PR https://github.com/livepeer/go-livepeer/pull/365/commits/f516851b3ae347d39464373aa7f15d7791bba233